src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -12,7 +12,7 @@ import useCreateChat from '../../../hooks/screen/useCreateChat';
 import TokenStorage, {ITokenEnum} from '../../../utils/storage/custom/tokenStorage';
 import {Context} from '../../../context';
 import {checkUserBlock} from '../../../service/profile';
-import {getChatName} from '../../../utils/string/StringUtils';
+import {getChatName, getOfficialAnonUsername} from '../../../utils/string/StringUtils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `getOfficialAnonUsername` function is a good addition. It will help in generating official anonymous usernames based on provided information.
```diff
- import {getChatName} from '../../../utils/string/StringUtils';
+ import {getChatName, getOfficialAnonUsername} from '../../../utils/string/StringUtils';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->